### ISSUE TITLE
UI improvement: Keep read items in feed list for Unread display mode until new feed selected

### ISSUE THREAD

**Issue by chrbayer** [jangernert/FeedReader#383]
Here is a proposal for an UI and usability improvement.

When in show only unread items do not remove read items as soon as they are read and the feed as soon as the last unread item has been read for the current feed. Remove the feed entry when a new feed is selected.

If I quickly go through the items in a feed, sometimes I realize a little bit to late that the last entry was worth reading but the way it is now, this entry is already gone and I have to switch to Show All to get it back and than go back to show only Unread items. This is not necessary with the proposed behavior.

**Comment by jangernert** [comment]
"It's not a bug but a feature"
Sounds to me like you should use "All Articles" ;)

**Comment by chrbayer** [comment]
No no, that is not the same. It should only show me the items that has been unread the time I selected the feed. FeedReader should only wait to hide the "old unread items" until a new feed is selected to allow quick re-selection of previous item. To allow my other proposed improvement it is important to stay in "Unread" display mode. The really great Reeder App for OS X for example behaves like this. I used this application a lot, so I'm quite sure that it is a real improvement :)
